lib/tags/assertions.js

```diff
@@ -83,6 +83,13 @@ module.exports = function yields (hexo, args) {
     </ul>`)
   }
 
+  const wrap = () => {
+    return render(`<ul>
+      <li><p>${cmd}, when its argument is a promise, will automatically {% url "retry" retry-ability %} until the promise resolves.</li>
```

Comment:
I don't think we should call it "retry" since we are NOT going to re-execute the promise, it is impossible. I think we can say "will automatically wait until the promise resolves".

We should also note that if the promise is rejected, `cy.wrap` will fail the test